docs/guides/designing-the-delta-format.md

```diff
@@ -27,7 +27,7 @@ var content = [
 ];
 ```
 
-If we want to add italics, underline, and other formats, we can add this to the main object, but it is cleaner to separate `text` from all of this so we organize formatting under one field, which we will name `attributes`.
+If we want to add italics, underline, and other formats; we can add this to the main object, but it is cleaner to separate `text` from all of this so we organize formatting under one field, which we will name `attributes`.
```

Comment:
The semicolon in this case indicates the list of elements is over.
http://writing.wisc.edu/Handbook/Semicolons.html
http://www.grammar-monster.com/lessons/semicolons_in_lists.htm